brd2json.ulp

```diff
@@ -444,7 +444,16 @@ board(B)
 					printf(tab(ilvl));
 					if(!firstattribute) { printf(","); } 
 					firstattribute = 0;
-					printf("\"%s\": \"%s\"\n", A.name, A.value);
+					string values[];
+					int len = strsplit(values, A.value, '"'); //"
```

Comment:
The `// "` comments are a workaround for my buggy ULP syntax highlighter. I forgot to remove them before pushing...